web/Weblate-server.qmd

```diff
@@ -205,26 +206,68 @@ rebase refs/remotes/origin/trunk: command returned error: 1
 Weblate will provide a suggestion on how to fix, but in short:
 
 1. SSH to the Weblate server
-2. Attach the Docker container:
+2. Make a backup on the Git repo before messing with the rebase process:
+
+    ```sh
+    tar -zcvf /root/base-r-gui-repo-$(date '+%Y-%m-%d').tar.gz \
+      /var/lib/docker/volumes/weblate-docker_weblate-data/_data/vcs/r-project/base-r-gui
+    ```
+
+3. Attach the Docker container (this image is offered by `weblate`):
 
     ```sh
     sudo docker exec -ti weblate-docker_weblate_1 bash
     ```
 
-3. Enter the git folder:
+4. Enter the git folder:
 
     ```sh
     cd app/data/vcs/r-project/base-r-gui
     ```
 
-4. Pull from SVN and start the rebase process to see the actual errors:
+    When looking for this folder outside of Docker, you can find it at
+
+    ```
+    /var/lib/docker/volumes/weblate-docker_weblate-data/_data/vcs/r-project/base-r-gui
+    ```
+
+5. Pull from SVN and start the rebase process to see the actual errors:
 
     ```sh
     git svn fetch
     git svn rebase
     ```
 
-5. Resolve the conflict via a text editor, then continue. Let's say `src/library/base/po/es.po` is affected:
+6. Optionally, if not yet done, install [helpers](https://github.com/JulienPalard/po3way) to make it easier to deal with the git conflicts (inside Docker):
```

Comment:
```suggestion
6. Optionally, if the rebase step fails, install [helpers](https://github.com/JulienPalard/po3way) to make it easier to deal with the git conflicts (inside Docker):
```

Reply:
here I tried to refer to the fact that these steps might not be needed, as already installed inside the Docker container. but when we update Weblate, a new Docker container will be created from a new Doker img, so the manual installs will be gone. so these steps are only required once after each update. could you pls help rephrase this? feel free to do direct edits :)

Reply:
I see, I misunderstood. Hopefully the new wording is good enough for future me :)